zipline/assets/assets.py

```diff
@@ -268,9 +268,10 @@ def lookup_future_chain(self, root_symbol, as_of_date):
         """
         root_symbol.upper()
         as_of_date = normalize_date(as_of_date)
-        return self._valid_contracts(root_symbol, as_of_date)
+        return self._valid_contracts(root_symbol, as_of_date, knowledge_date)
 
-    def lookup_future_in_chain(self, root_symbol, as_of_date, contract_num=0):
+    def lookup_future_in_chain(self, root_symbol, as_of_date, knowledge_date,
```

Comment:
Not sure if we should keep this around, since the FutureChain class should only have to call `lookup_future_chain`, and then work from that.